packages/api/cms-api/src/dam/files/files.service.ts

```diff
@@ -559,8 +559,7 @@ export class FilesService {
     async createFileUrl(file: FileInterface, previewDamUrls?: boolean): Promise<string> {
         const filename = parse(file.name).name;
 
-        // Use CDN url only if available and not in preview as preview requires auth
-        const baseUrl = [this.config.cdnEnabled && !previewDamUrls ? `${this.config.cdnDomain}/files` : this.config.filesBaseUrl];
+        const baseUrl = [this.config.filesBaseUrl];
```

Comment:
This also removes the previewDamUrls check. Meaning there is no possibility to switch between the privateApiUrl (Authproxy) and publicApiUrl. Will this be readded in a follow-up PR? Or will this be handled differently?

Reply:
Yes, you are absolutely right. I've tried to make as few changes as possible, but it seems that more changes are needed.

I've addressed this in 5e707ebbce4adf879fccc35bda932404608d13fd

Please have another look. Once we agree on a solution, I will update the docs and migration guide accordingly.